docs/user/revoke-provider-access.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Revoke Provider Access"
+description: "Revokes the access token for a given provider."
+api: "DELETE /user/revoke"
+---
+
+## Query Params
+
+<ParamField query='userId' type='string' required>
+  The Metriport user ID to revoke access for.
+</ParamField>
+
+<ParamField query='provider' type='string' required>
+  The provider you would like to revoke access to.
+</ParamField>
+
+## Response
+
+<ResponseField name='success' type='boolean' required>
+  Whether operation was successful or not.
+</ResponseField>
+
+```json
+{
+  "success": true
+}
+```
```

Comment:
My 2 cents would be to just return 200 on success, and some other code if not successful.

Personally as a dev, I dislike when APIs make me check 200 and then a success bool as well. 200 == success, and normally API clients throw an error on other return codes, so adding `  "success": true` doesn't give the caller any more info.

@leite08 thoughts?

Reply:
I had it like that originally but i didn't know if that was the best approach so i tried to see what others did and some apis return a success object. Its the same to me though i have no preference. Id like to see what @leite08  has to say

Reply:
Agreed!

This ties to the API response payload that we defined on `default-error-handler.ts`, based on [RFC 7807](https://www.rfc-editor.org/rfc/rfc7807). Actually, the `default-error-handler` actually needs some improvements, as we're transforming the  RFC payload into JSON without typing it first, so there's no protection there in compile time - but that's the idea.